core/src/main/java/com/datastrato/gravitino/listener/api/info/partitions/IdentityPartitionInfo.java

```diff
@@ -0,0 +1,95 @@
+/*
```

Comment:
seems only to move these files to the correct package.

Reply:
Done, fixed build error